src/Services/Filters.js

```diff
@@ -44,11 +44,21 @@ export const CLUSTER_RULES_INITIAL_STATE = {
   text: '',
 };
 
+export const WORKLOADS_LIST_INITIAL_STATE = {
+  limit: 50,
+  offset: 0,
+  sortIndex: CLUSTERS_TABLE_CELL_LAST_SEEN,
```

Comment:
Should we use the constant for the workloads table instead?

Reply:
Why renaming? I think creating a new one is the right way since these are two separate tables.

Reply:
Aaaaaaaaah! You mean for the next table! Yes, there will be a constant for it as well, not in this PR though :D

Reply:
I'm stupid...for some reason, I thought you were talking about the object itself...